src/components/landing-page/WhyTranscripts.tsx

```diff
@@ -0,0 +1,109 @@
+import Wrapper from "../layout/Wrapper";
+import SuggestModal from "./SuggestionModal";
+
+const WhyTranscripts = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  React.useEffect(() => {
+    document.body.classList.toggle("overflow-hidden", isOpen);
+  }, [isOpen]);
+
+  return (
+    <Wrapper className='py-[104px] text-black flex flex-col gap-[104px] max-md:py-16 max-md:gap-12 bg-gray-custom-900'>
+      <div className='flex flex-col items-center justify-center gap-12 max-md:gap-4'>
+        <p className='text-purple-custom-200 bg-purple-custom-100 py-2 px-6 rounded-full max-md:text-sm max-md:py-[2px] max-md:px-3.5'>
+          WHY TRANSCRIPTS?
+        </p>
+        <h2 className='2xl:text-[72px] text-[64px] leading-[86px] text-center font-medium max-w-[1500px] max-xl:text-5xl max-xl:leading-[130%] max-[953px]:text-[38px] max-[953px]:leading-[130%] max-md:text-[40px] max-sm:leading-[56.4px] max-md:max-w-[368px]'>
+          A Historical Archive for Knowledge Preservation and Propagation
+        </h2>
+      </div>
+
+      <div className='flex justify-between items-center gap-[69px] max-md:flex-col max-lg:gap-8 max-md:gap-6'>
+        <section className='max-w-[50%] flex flex-col max-md:items-start max-md:max-w-full'>
+          <h4 className='text-5xl font-medium max-lg:text-3xl max-md:text-2xl leading-[130%]'>Transcripts Since 2014</h4>
+          <p className='pt-6 pb-8 text-xl max-lg:text-base max-md:pb-[18px] max-md:pt-[18px]'>
+            Bryan Bishop{" "}
+            <span>
+              <Link href='https://github.com/kanzure' target='_blank' className='text-blue-custom-100 underline'>
+                (@kanzure)
+              </Link>
+            </span>{" "}
+            manually transcribed ~900 transcripts over the years, making them publicly available at{" "}
+            <span>
+              <Link href='https://github.com/kanzure/diyhpluswiki' target='_blank' className='text-blue-custom-100 underline'>
+                diyhpluswiki
+              </Link>
+            </span>
+            . <br /> <br /> Bitcoin Transcripts builds on kanzure's foundational work and is now actively maintained and enriched by the community.
+          </p>
+          <div className='flex items-center justify-center max-md:w-full'>
+            <Link
+              href='/transcripts'
+              className='text-xl bg-orange-custom-100 text-white py-6 rounded-full flex items-center md:w-fit px-32 whitespace-nowrap justify-center h-20 max-xl:h-[72px] max-lg:h-16 max-md:h-14 max-lg:w-full max-md:w-full font-semibold text-nowrap max-lg:text-lg max-md:text-base max-lg:px-16 max-md:px-8'
+            >
+              Explore Bitcoin Transcripts
+            </Link>
+          </div>
+        </section>
+        <section className='max-w-[50%] max-h-[587px] max-md:max-w-full max-md:max-h-[500px] max-[400px]:max-h-[261px]'>
+          <Image src={userTranscribing} loading='lazy' alt='user transcribing' className='w-full h-full' />
```

Comment:
@IgboPharaoh we are still missing the GIF here. You can find it on figma.

Reply:
We have not added the gif because the file is large and tobi is still looking for ways to compress it further or possibly reduce the duration of the gif to reduce it's size. It's currently at 12mb and that is a whole lot.
From the last meeting we had, Art said we can move forward with this since that hasn't been finalised yet.

Reply:
Oh, you are right! totally forgot that. But maybe we can use a frame from the gif so that we actually have Bryan Bishop typing?

Reply:
I think that can be done, I've tried it out before and the image quality wasn't impressive which was why I decided to wait for the gif.
But I think @Emmanuel-Develops  can add it here and try it out

Reply:
As Solomon said, the image is low quality. The community can be very nitpicky about things like this, but for me its fine either ways.

<img width="1745" alt="Screenshot 2024-09-12 at 11 49 53 AM" src="https://github.com/user-attachments/assets/0f079fd6-a8a9-4930-a69f-4c14ed635cb5">
cc: @kouloumos

Reply:
We took the video from here:https://x.com/peterktodd/status/926942016990208000

When I download the video, it's 3.2MB

https://github.com/user-attachments/assets/2bb49ec9-e63a-4709-a310-da7385a0ba8a

It doesn't sounds right to me, but the GIF from the video is much larger (10X) than the video. 

@IgboPharaoh @Emmanuel-Develops what do you thinking about using the 3.2MB video autoplay and in a loop? Isn't the same?

Reply:
Seems that is how gif conversion works I have tried different reduction and they all have large file sizes.
This has been a bottleneck and we should tackle this outside of merging staging to prod.
Issue: https://github.com/bitcointranscripts/registry/issues/25

cc: @kouloumos @IgboPharaoh. Here is the PR to fix the hash and swap the image https://github.com/bitcointranscripts/registry/pull/24